index.html

```diff
@@ -13,7 +13,7 @@
     <div class="alert alert-dark alert-dismissible mb-0" role="alert">
       <!-- Only this <p> element should be changed in an alert -->
       <p class="text-center">
-	      JuliaCon is <a href="https://live.juliacon.org/live">live now!</a> JuliaCon 2020 is free and live now: <a href="https://juliacon.org/2020/tickets/">Register here</a>!
+	      JuliaCon is <a href="https://live.juliacon.org/live">live now!</a> <a href="https://juliacon.org/2020/tickets/">Register here</a>. Julia Community Prizes<a href="https://juliacon.org/2020/prize/">announced</a>! 
```

Comment:
Missing space between "Prizes" and "announced" (as clearly seen in the preview!).